src/planscape/planning/serializers.py

```diff
@@ -0,0 +1,18 @@
+from conditions.models import BaseCondition, Condition
+from rest_framework import serializers
+from rest_framework.serializers import IntegerField
+from rest_framework_gis import serializers as gis_serializers
+
+from .models import (PlanningArea, Scenario)
+
+
+class PlanningAreaSerializer(gis_serializers.GeoFeatureModelSerializer):
+    class Meta:
+        fields = ("id", "user", "name", "region_name")
+        model = PlanningArea
+        geo_field = "geometry"
+
+class ScenarioSerializer(serializers.ModelSerializer):
```

Comment:
yea, I kept this one in before I decided to just to PlanningAreas; I can remove it if needed.